### DISCUSSION TITLE
Q_Li and Q_p error

### DISCUSSION THREAD

**Discussion by cyasing** [pybamm-team/PyBaMM#2819] [Q&A] [answered]
I need to know what this error means. It happens with custom parameters: 
`
UserWarning: Q_Li=1.1560 Ah is greater than Q_p=0.9326 Ah.
  warnings.warn(f"Q_Li={Q_Li:.4f} Ah is greater than Q_p={Q_p:.4f} Ah.")
`

**Comment by valentinsulzer** [comment, marked as answer]
It's not an error. It's a warning. It's impossible to have more cyclable lithium than the cathode capacity, since the anode starts out fully delithiated. See this paper https://www.frontiersin.org/articles/10.3389/fenrg.2023.1087269/full 

You need to change the initial concentrations to change the total cyclable lithium. There's a recent discussion somewhere here on that

**Reply by cyasing** [reply]
Understood. Can you tell me which of the following parameters might create such warning? It's updated into parameter set Ai2020

- "Positive electrode thickness [m]" : 42e-6,
- "Positive electrode active material volume fraction" : 0.29,
- "Positive particle radius [m]" : 4.85e-6,

- "Negative electrode thickness [m]" : 55e-6,
- "Negative electrode active material volume fraction" : 0.39,
-  "Negative particle radius [m]" : 13e-6,

-  "Cell cooling surface area [m2]" : 0.0572, # 26 x 22 cm 
-  "Nominal cell capacity [A.h]": 57.5,
-  "Electrode height [m]": 0.26,
-  "Electrode width [m]": 0.22,
-  "Cell volume [m3]": 26 * 22 * 0.8 * 1e-6,
-  "Upper voltage cut-off [V]": 4.7,

**Reply by tGisbertz** [reply]
I think all parameters that influence the electrode capacities directly (electrode thickness, active material volume fraction, electrode height/width) or are among the parameters for the esoh calculation (like voltage curves/cut-offs, nominal capacity) could be the reason for the warning.
